### PR TITLE
Billing exporter

### PR DESCRIPTION
**Pull request by saliceti**
## What
Scrape and display billing data

## How to review
Dashboard deployed to https://grafana-bat-qa.london.cloudapps.digital/
image

### CODE REVIEW

File: prometheus/templates/prometheus.yml.tmpl [hunk 1]
```diff
@@ -31,6 +31,14 @@ scrape_configs:
       port: ${app.port}
 %{ endfor ~}
 
+%{ if billing_exporter_endpoint != "" }
```
Comment (vigneshmsft) [inline prometheus/templates/prometheus.yml.tmpl:34]: why not include it as part of the existing var.exporters? and have a default scarpe_internval for others?

File: prometheus_all/resources.tf [hunk 2]
```diff
@@ -22,6 +22,15 @@ module "postgres_prometheus_exporter" {
   postgres_service_instance = each.key
 }
 
+module "billing_prometheus_exporter" {
+  count                    = var.enable_billing_exporter ? 1 : 0
```
Comment (vigneshmsft) [inline prometheus_all/resources.tf:26]: include as part of enabled modules?

File: prometheus/templates/prometheus.yml.tmpl (hunk 1 above)
Comment (saliceti) [reply prometheus/templates/prometheus.yml.tmpl]: done

File: prometheus_all/resources.tf (hunk 2 above)
Comment (saliceti) [reply prometheus_all/resources.tf]: done

**Comment by saliceti** [comment]
[Cost monitor on paas](https://trello.com/c/2Rxy8Vjz/186-cost-monitor-on-paas)

**Pull request merged**